vendor/Makefile

```diff
@@ -75,7 +74,7 @@ clean:
 	$(RM) $(SAXON_HE_ZIP)
 	$(RM) $(PAGE_SCHEMA_REPO)
 	$(RM) $(ALTO2PAGE_DIR)
-	$(RM) $(ALTO2PAGE_ZIP)
+	$(RM) $(ALTO2PAGE-ZIP)
```

Comment:
That looks like a typo now with the `-` instead of `_`.

Reply:
It is. I'll fix.